test/parallel_api/algorithm/alg.sorting/sort.pass.cpp

```diff
@@ -200,29 +252,110 @@ struct test_sort_with_compare
         for (size_t i = 0; i < n; ++i, ++expected_first, ++tmp_first)
```

Comment:
There is EXPECT_EQ_N in the test utilities - see utils.h

Reply:
Fixed

Reply:
Fixed